### PR TITLE
devenv: Update several dashes to schemaVersion 36

### PR DESCRIPTION
**Pull request by sdboyer**
**What this PR does / why we need it**:

This updates a handful of the dashboards in devenv to schemaVersion 36. That's the minimum version the thema schema will test against, so this gives us a better regression/check suite.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by sdboyer** [comment]
i'm not sure what's causing the downstream enterprise build problem, but on the e2e test i can just revert the one changed dashboard here.

**Review by ryantxu — APPROVED** [review]

File: devenv/dev-dashboards/panel-candlestick/candlestick.json [hunk 1]
```diff
@@ -21,10 +24,15 @@
...
   "fiscalYearStartMonth": 0,
   "graphTooltip": 2,
+  "id": 93,
   "links": [],
   "liveNow": false,
   "panels": [
     {
+      "datasource": {
+        "type": "testdata",
+        "uid": "3fuUBUNGz"
```
Comment (torkelo) [inline devenv/dev-dashboards/panel-candlestick/candlestick.json:34]: @sdboyer 
Where did this come from? It is not the default data source, the uid is wrong

File: devenv/dev-dashboards/panel-geomap/geomap_multi-layers.json [hunk 2]
```diff
@@ -21,23 +24,33 @@
...
-  "id": 157,
+  "id": 88,
   "links": [],
   "liveNow": false,
   "panels": [
     {
+      "datasource": {
+        "type": "testdata",
+        "uid": "3fuUBUNGz"
+      },
```
Comment (torkelo) [inline devenv/dev-dashboards/panel-geomap/geomap_multi-layers.json:35]: wrong uid here as well

File: devenv/dev-dashboards/panel-candlestick/candlestick.json (hunk 1 above)
Comment (sdboyer) [reply devenv/dev-dashboards/panel-candlestick/candlestick.json:34]: sorry, IIRC i just did an export via the UI and dumped it to disk - that's what popped out

**Pull request merged**